<ui version="4.0" >
 <class>RVFileView</class>
 <widget class="QWidget" name="RVFileView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="tree" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="directory" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QToolButton" name="reload_button" >
       <property name="text" >
        <string>Reload File List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="save_button" >
       <property name="text" >
        <string>Save Experiment...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="open_button" >
       <property name="text" >
        <string>Open Experiment...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
